package.json

```diff
@@ -34,6 +34,7 @@
     "eslint-plugin-nuxt": ">=0.4.2",
     "eslint-plugin-prettier": "^3.1.2",
     "jest": "^25.1.0",
+    "nodemon": "^2.0.2",
```

Comment:
This worked on my side because I have nodemon installed globally thank you for pointing it out